index.d.ts

```diff
@@ -1,12 +1,49 @@
-import { Parser } from 'acorn' 
+import * as acorn from 'acorn';
 
-declare const jsx: (options?: jsx.Options) => (BaseParser: typeof Parser) => typeof Parser;
+export const jsx: (options?: jsx.Options) => (BaseParser: typeof acorn.Parser) => typeof AcornJsxParser
```

Comment:
This declares `jsx` is available as a named export, which is incorrect.

Reply:
Makes sense, and I've added a commit to address.